modules/ribbit/http.py

```diff
@@ -53,25 +64,7 @@ async def config_list(req):
             out["domain"] = DOMAIN_NAMES[domain]
             if not key_info.protected:
                 out["value"] = value
-
+        
         return json.dumps(ret), 200, {"Content-Type": "application/json"}
-
-    @app.patch("/api/config")
-    async def config_set(req):
-        values = req.json
-
-        if not isinstance(values, dict):
-            raise HTTPException(400)
-
-        try:
-            registry.config.set(DOMAIN_LOCAL, values)
-            return "{}", 201, {"Content-Type": "application/json"}
-
-        except ValueError as exc:
-            return (
-                json.dumps({"error": str(exc)}),
-                400,
-                {"Content-Type": "application/json"},
-            )
-
```

Comment:
Why are you removing this?

Reply:
Hmm, I can't seem to find where this route is being used. What is this used for?

Reply:
This is the API to mutate the config.